app/assets/javascripts/uploadcare/widget/tabs/preview-tab.coffee

```diff
@@ -77,6 +86,24 @@ uploadcare.namespace 'widget.tabs', (ns) ->
             @__setState('image', {file: false})
             @element('image').attr('src', src)
             @initImage(size)
+            df.resolve()
+      .fail () =>
+        return df.reject('fail')
+
+      df.promise()
+
+    __tryToLoadVideoPreview: (file, blob) =>
+      if (
+        file.state() != 'pending' or
+          not blob.size
```

Comment:
wrong Indention.

There also should be the check is `URL` available. For images it also done in `drawFileToCanvas()`.